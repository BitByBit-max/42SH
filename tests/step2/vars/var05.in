commit_message="wip(testsuite):  Work in progress!"
followtags='deploy-42sh'
options='--follow-tags --force'

echo "git add --force .";
echo "git commit -m $commit_message";
echo "git tag -a $followtags -m $followtags";
echo "git push $options";
