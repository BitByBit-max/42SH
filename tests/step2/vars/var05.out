git add --force .
git commit -m wip(testsuite):  Work in progress!
git tag -a deploy-42sh -m deploy-42sh
git push --follow-tags --force
